affected_purls:
- pkg:pypi/oauthenticator@0.12.0
- pkg:pypi/oauthenticator@0.12.1
aliases:
- CVE-2020-26250
- GHSA-384w-5v3f-q499
fixed_purl:
- pkg:pypi/oauthenticator@0.12.2
references:
- id: 153146
  reference_id: ''
  url: https://github.com/jupyterhub/oauthenticator/blob/master/docs/source/changelog.md#0122---2020-11-30
- id: 153145
  reference_id: ''
  url: https://github.com/jupyterhub/oauthenticator/commit/a4aac191c16cf6281f3d346615aefa75702b02d7
- id: 153147
  reference_id: ''
  url: https://jupyterhub.readthedocs.io/en/1.2.2/getting-started/authenticators-users-basics.html#add-or-remove-users-from-the-hub
- id: 153148
  reference_id: CVE-2020-26250
  url: https://nvd.nist.gov/vuln/detail/CVE-2020-26250
- id: 153149
  reference_id: GHSA-384w-5v3f-q499
  url: https://github.com/advisories/GHSA-384w-5v3f-q499
- id: 153144
  reference_id: GHSA-384w-5v3f-q499
  url: https://github.com/jupyterhub/oauthenticator/security/advisories/GHSA-384w-5v3f-q499
severities:
- id: 21893
  reference_id: 153149
  scoring_elements: ''
  scoring_system: cvssv3.1_qr
  value: HIGH
- id: 21892
  reference_id: 153144
  scoring_elements: ''
  scoring_system: cvssv3.1_qr
  value: HIGH
summary: Base class whitelist configuration ignored in OAuthenticator
vulnerability_id: VCID-utxv-fu4e-aaae
weaknesses: []
